homeassistant/components/camera/synology.py

```diff
@@ -39,22 +39,26 @@
 
 
 @asyncio.coroutine
-def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+def async_setup_platform(hass, config, async_add_devices,
+                         discovery_info=None):
     """Set up a Synology IP Camera."""
+    url = config.get(CONF_URL)
+    username = config.get(CONF_USERNAME)
+    password = config.get(CONF_PASSWORD)
     verify_ssl = config.get(CONF_VERIFY_SSL)
     timeout = config.get(CONF_TIMEOUT)
 
     try:
         from synology.surveillance_station import SurveillanceStation
         surveillance = SurveillanceStation(
-            config.get(CONF_URL),
-            config.get(CONF_USERNAME),
-            config.get(CONF_PASSWORD),
+            url,
+            username,
+            password,
             verify_ssl=verify_ssl,
             timeout=timeout
         )
     except (requests.exceptions.RequestException, ValueError):
-        _LOGGER.exception("Error when initializing SurveillanceStation")
+        _LOGGER.exception("Error initializing SurveillanceStation")
```

Comment:
This should be `_LOGGER.error`, it's an expected exception. Also, what can cause a ValueError?